raiden/api/rest.py

```diff
@@ -561,6 +561,20 @@ def open(
             token_address=to_checksum_address(token_address),
             settle_timeout=settle_timeout,
         )
+        token = self.raiden_api.raiden.chain.token(token_address)
+        balance = token.balance_of(self.raiden_api.raiden.address)
+
+        if total_deposit is not None and total_deposit > balance:
+           error_msg = 'Not enough balance to deposit. {} Available={} Needed={}'.format(
```

Comment:
fix indentation level here to be 4 spaces. As I said before you can check that this is okay before committing by running flake8 on your code.